zerver/tests/test_realm_export.py

```diff
@@ -323,3 +324,30 @@ def test_upload_and_message_limit(self) -> None:
             result,
             f"Please request a manual export from {settings.ZULIP_ADMINISTRATOR}.",
         )
+
+    def test_get_users_export_consents(self) -> None:
+        admin = self.example_user("iago")
+        self.login_user(admin)
+
+        # By default, export consent is set to False.
+        rows = UserProfile.objects.filter(realm=admin.realm, is_active=True, is_bot=False).values(
+            "id", "allow_private_data_export"
+        )
+        for row in rows:
+            self.assertFalse(row["allow_private_data_export"])
```

Comment:
This could be better written as just `self.assertFalse(UserProfile.objects.filter(realm=admin.realm, is_active=True, is_bot=False, allow_private_data_export=True).exists()`.